src/main/webapp/WEB-INF/tags/fp/dvPayment.tag

```diff
@@ -332,28 +333,42 @@
-            </tr>
-            <tr>
-                <th align=right valign=middle class="bord-l-b"><div align="right">
-                    <kul:htmlAttributeLabel attributeEntry="${dvAttributes.invoiceNumber}" /></div></th>
-                <td align=left valign=middle class="datacell">
-                    <kul:htmlControlAttribute property="document.invoiceNumber"
-                                              attributeEntry="${dvAttributes.invoiceNumber}"
-                                              readOnly="${!fullEntryMode && !paymentHandlingEntryMode}" /></td>
-            </tr>
-            <tr>
-                <th align=right valign=middle class="bord-l-b"><div align="right">
-                    <kul:htmlAttributeLabel attributeEntry="${dvAttributes.invoiceDate}" /></div></th>
-                <td align=left valign=middle class="datacell">
-                    <kul:htmlControlAttribute property="document.invoiceDate"
-                                              attributeEntry="${dvAttributes.invoiceDate}"
-                                              readOnly="${!fullEntryMode && !paymentHandlingEntryMode}" /></td>
-            </tr>
-            <tr>
+            			</tr>
+
+	            		<tr>
+	                		<th align=right valign=middle class="bord-l-b"><div align="right">
+	                    		<kul:htmlAttributeLabel attributeEntry="${dvAttributes.invoiceNumber}" /></div></th>
+	                		<td align=left valign=middle class="datacell">
+	                    		<kul:htmlControlAttribute property="document.invoiceNumber"
+	                                              attributeEntry="${dvAttributes.invoiceNumber}"
+	                                              readOnly="${!fullEntryMode && !paymentHandlingEntryMode}" /></td>
+	            		</tr>
+	            		<tr>
+	                		<th align=right valign=middle class="bord-l-b"><div align="right">
+	                    		<kul:htmlAttributeLabel attributeEntry="${dvAttributes.invoiceDate}" /></div></th>
+	                		<td align=left valign=middle class="datacell">
+	                    		<kul:htmlControlAttribute property="document.invoiceDate"
+	                                              attributeEntry="${dvAttributes.invoiceDate}"
+	                                              readOnly="${!fullEntryMode && !paymentHandlingEntryMode}" /></td>
+	            		</tr>
+	            		 <tr>
+            		</c:when>
+            		<c:otherwise>
+	        			<tr>
+-              			<th scope="row"><div align="right"><kul:htmlAttributeLabel attributeEntry="${dvAttributes.disbVchrCheckStubText}"/></div></th>
+-              			<td><kul:htmlControlAttribute attributeEntry="${dvAttributes.disbVchrCheckStubText}" 
+							property="document.disbVchrCheckStubText" readOnly="${!fullEntryMode && !paymentHandlingEntryMode}"/></td>
```

Comment:
Is there a formatting issue that requires the "disbVchrCheckStubText" field to exclude the rowspan setup when the invoice fields are not visible? If so, then are changes also needed to field in any of the other table rows?

Reply:
After running this code locally, things seem to be rendering properly. It looks like the extra rowspan setup allows for keeping the two new invoice fields in the same column.